automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -29,7 +32,20 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   private val modalConfirmDeleteButton = Button("modal-confirm-delete-button")
   private val snapshotRedactedLabel = Label("snapshot-redacted-title")
 
-  def clickLaunchAnalysis(): Unit = openLaunchAnalysisModalButton.doClick()
+  def clickLaunchAnalysis(): Unit = {
+    // retry click if no Modal found. click sometimes weirdly doesn't work
+    val clickAgain: Try[Element] = Try{
+      openLaunchAnalysisModalButton.doClick()
+      // after click, expect to find either a Message or Analysis Modal
+      find(CssSelectorQuery("body.broadinstitute-modal-open")).get
+    }
+    clickAgain match {
+      case Failure(e) =>
+        logger.warn("Attempting to re-click button \"Launch Analysis...\"")
+        openLaunchAnalysisModalButton.doClick()
+      case Success(some) =>
+    }
+  }
```

Comment:
I don't like this, but I'm okay with this as a hack until we figure out the real problem. Can you create a ticket and link to it in a comment?

Reply:
revert. I feel this is being addressed with adding a short sleep as a hack in  `doClick.scrollToVisible`